api/browser.ts

```diff
@@ -55,6 +63,27 @@ export class KastleBrowserAPI {
     return await this.receiveMessageWithTimeout(requestId);
   }
 
+  async disconnect(): Promise<void> {}
+
+  async request(method: string, args?: unknown): Promise<any> {
+    const requestId = uuid();
+    const action = {
+      "get-account": Action.GET_ACCOUNT,
+      "get-network": Action.GET_NETWORK,
+      "sign-tx": Action.SIGN_TX,
+      "sign-and-broadcast-tx": Action.SIGN_AND_BROADCAST_TX,
```

Comment:
Would it be better to have `kaspa:` prefix?

Reply:
I agree with the prefix, but it's not wise to mix up cases.
I like snake case, but that's an opinion.

Reply:
3ee36a59e314208f24a46de7206e567cae2a8ca1